vpr/SRC/route/route_timing.c

```diff
@@ -300,20 +333,31 @@ bool try_timing_driven_route(struct s_router_opts router_opts,
 // profiling per iteration functions
 #ifdef PROFILE
 void time_on_fanout_analysis() {
+	vpr_printf_info("%d entire net rerouted, %d entire trees pruned, %d partially rerouted\n", 
+		entire_net_rerouted, entire_tree_pruned, part_tree_preserved);

```

Comment:
output should probably say what "entire_tree_pruned" is (net routing unchanged).